packages/playground/windows/playground/HostingPane.xaml.cpp

```diff
@@ -221,6 +221,7 @@ std::shared_ptr<react::uwp::IReactInstance> HostingPane::getInstance() {
     settings.UseLiveReload = x_UseLiveReloadCheckBox->IsChecked->Value;
     settings.UseDirectDebugger = x_UseDirectDebuggerCheckBox->IsChecked->Value;
     settings.DebuggerBreakOnNextLine = x_BreakOnFirstLineCheckBox->IsChecked->Value;
+    settings.DebuggerPort = static_cast<uint16_t>(std::stoi(std::wstring(x_DebuggerPort->Text->Data())));
```

Comment:
>std::stoi [](start = 50, length = 9)

This API may throw. Would you consider to have a utility function that is shared between UWP and Win32?

Reply:
Isn't this partially what `std::from_chars` was meant to fix?